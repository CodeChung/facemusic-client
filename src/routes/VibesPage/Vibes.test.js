import React from 'react';
import ReactDOM from 'react-dom';
import Preferences from './Vibes';
import { BrowserRouter } from 'react-router-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Preferences Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter><Preferences /></BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('displays a proper PreferencesPage', () => {
    const wrapper = shallow(<BrowserRouter><Preferences /></BrowserRouter>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})