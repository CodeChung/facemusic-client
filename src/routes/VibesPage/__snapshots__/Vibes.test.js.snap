// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preferences Component displays a proper PreferencesPage 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Preferences />
</Router>
`;
